src/ExpressReceiver.ts

```diff
@@ -28,9 +30,10 @@ export default class ExpressReceiver extends EventEmitter implements Receiver {
 
   private server: Server;
 
-  constructor ({
+  constructor({
     signingSecret = '',
-    endpoints = { events: '/slack/events' },
+    logger = new ConsoleLogger(),
```

Comment:
Can `App` pass in a `logger` when initializing this object? That way the `logLevel` is respected all the way down.

Reply:
Thank you for pointing out this. I should have been more careful about it... Anyway, I've updated the branch to fix this.